### ISSUE TITLE
Sequence returns empty response and a 200 status code after all responses are exhausted

### ISSUE THREAD

**Issue by pavellishin** [dreamhead/moco#176]
https://github.com/dreamhead/moco/blob/master/moco-doc/apis.md#sequence

The documentation mentions nothing about what happens if you make more requests to an endpoint than there are responses defined, but based on what we saw in a test, it seems to respond with an empty string, and a 200 status code.

This is non-obvious at first, and made it unnecessarily difficult to debug why a test was failing.

I think a 501 response with a human-readable body would be more reasonable (or a 400, or 410, or maybe a 429.)

**Comment by pavellishin** [comment]
Although, looking at the code, it looks like it ought to return the last response repeatedly, I wonder if we're suing an older version: https://github.com/dreamhead/moco/blob/master/moco-core/src/main/java/com/github/dreamhead/moco/handler/SequenceHandler.java#L33

**Comment by dreamhead** [comment]
You're right. The last response will be returned. If it does not work like that, it's supposed to be a bug. Please provide your configuration, I'll check it.

**Comment by pavellishin** [comment]
I'll try to get a working example of what I'm seeing later today
